automatic/ant-renamer/ant-renamer.nuspec

```diff
@@ -17,7 +17,7 @@ Ant Renamer is a free (really free, i. e. the source code is available) program
     <releaseNotes>http://www.antp.be/software/renamer/history</releaseNotes>
     <copyright>Antoine Potten</copyright>
     <tags>renamer regex admin</tags>
-    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/ant-renamer</packageSourceUrl>
```

Comment:
@AdmiringWorm if this change is going to be made, we should do it on all of them :-)